tests/py_test.py

```diff
@@ -540,15 +583,26 @@ def asserter(stmt):
         asserter(util.GetBufferIdentifier(buf, offset, size_prefixed=sizePrefix) == file_identifier)
         asserter(util.BufferHasIdentifier(buf, offset, file_identifier=file_identifier, size_prefixed=sizePrefix))
         asserter(MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset, size_prefixed=sizePrefix))
+    else:
+        # test may fail if size_prefix equals the Identifier, but this is very unlikely
+        asserter(not MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
+
+    oldoffset = offset
     if sizePrefix:
         size = util.GetSizePrefix(buf, offset)
         asserter(size == len(buf[offset:])-4)
         buf, offset = util.RemoveSizePrefix(buf, offset)
     if file_identifier:
+        # test again after removal of size_prefix
+        asserter(util.GetBufferIdentifier(buf, offset) == file_identifier)
+        asserter(util.BufferHasIdentifier(buf, offset, file_identifier=file_identifier))
         asserter(MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
     else:
+        # test may fail if vtable offset equals the Identifier, but this is very unlikely
         asserter(not MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
     monster = MyGame.Example.Monster.Monster.GetRootAs(buf, offset)
+    # wrap moster objects _tab with one that verifies alignment
+    monster = TestProxyTable.wrapMonster(monster, oldoffset)
```

Comment:
I could replace this with

    def wrapGet(monster, rootOffset):
        oldGet = monster._tab.Get
        def Get(self, flags, off):
            """ Assert that an offset is properly aligned prior to Table.Get """
            
            N.enforce_number(off, N.UOffsetTFlags)

            relativeOffset = off - rootOffset
            misalignment = relativeOffset % flags.bytewidth
            if misalignment != 0:
                raise AssertionError("Address %d is not alligned to %d. Off by %d" \
                   % (relativeOffset, flags.bytewidth, misalignment))

            return oldGet(self, flags, off)
		
		monster._tab.Get =  Get
		return monster
		
    monster = wrapGet(monster, oldoffset)